tooling/camel-kafka-connector-docs-maven-plugin/pom.xml

```diff
@@ -158,4 +158,20 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
You don't need to do this in tooling. There is the MOJO invocation in connectors/pom.xml and you should set the option there eventually, but they already have default value, so this seems to be useless